apps/rush-lib/src/scripts/install-run.ts

```diff
@@ -452,9 +452,16 @@ export function installAndRun(
   const originalEnvPath: string = process.env.PATH || '';
   let result: childProcess.SpawnSyncReturns<Buffer>;
   try {
+    // Node.js on Windows can not spawn a file when the path has a space on it
+    // unless the path gets wrapped in a cmd friendly way and shell mode is used
+    const shouldUseShell: boolean = binPath.includes(' ') && os.platform() === 'win32';
+    const scapedBinPath: string = shouldUseShell ? `"${binPath}"` : binPath;
```

Comment:
Is 'scaped' a verb? Did you mean 'escaped'? Is wrapping in quotes really considered 'escaping'? What even are words?

Reply:
I can rename it, any preference for the name?

Reply:
Maybe `platformBinPath` to emphasize that it's modified due to platform specific reasons?

Reply:
Updated with the proposed name